libs/infra-tracing/src/lib/datadog-tracer.spec.ts

```diff
@@ -33,33 +33,4 @@ describe('datadogTrace', () => {
       expect(ddtrace.shouldMask(500)).toBeFalsy()
     })
   })
-
-  describe('rewriteUrl', () => {
-    it('should remove valid nationalid from url', () => {
-      const ddtrace = require('./datadog-tracer') //eslint-disable-line
-
-      expect(
-        ddtrace.rewriteUrl(
-          `http://island.is/path/to/${generatePerson(new Date(0))}`,
-        ),
-      ).toEqual('http://island.is/path/to/--MASKED--')
-      expect(
-        ddtrace.rewriteUrl(
-          `http://island.is/path/to/${generatePerson(
-            new Date(0),
-          )}/something/more`,
-        ),
-      ).toEqual('http://island.is/path/to/--MASKED--/something/more')
-    })
-    it('should not mask if not valid nationalid', () => {
-      process.env.DD_PII_MASKING_DISABLED_ON_FAILURE = 'true'
-      const ddtrace = require('./datadog-tracer') //eslint-disable-line
-
-      ;[
-        'http://island.is/path/to/0000000000',
-        'http://island.is/path/to/1234567890/',
-        'http://island.is/path/to/5864239878/boiah',
-      ].forEach((url) => expect(ddtrace.rewriteUrl(url)).toEqual(url))
-    })
-  })
```

Comment:
Why remove this?

Reply:
There are currently two different functions that mask nationalIds in the codebase (logging library and datadog-tracer library) and I needed to create the third one.

So instead I created the library `libs/shared/pii` which has one function `maskNationalId` which I use in all three places.